jobs/load-from-openmrs-hiv-sites.yml

```diff
@@ -0,0 +1,11 @@
+type: "iterating-job"
+description: "Refreshing OpenMRS Data for all sites"
+configuration:
```

Comment:
Configuration should change to reflect this as a job-pipeline.

First job = load-hiv-data-from-openmrs.yml" (with siteName, partitionNum, dataset as variables)
Second job = "load-warehouse-hiv-analysis-tables.yml"